Content.Server/Electrocution/ElectrocutionSystem.cs

```diff
@@ -323,6 +323,10 @@ private void OnElectrifiedInteractUsing(EntityUid uid, ElectrifiedComponent elec
         var electrocutionNode = Comp<NodeContainerComponent>(electrocutionEntity).GetNode<ElectrocutionNode>("electrocution");
         var electrocutionComponent = Comp<ElectrocutionComponent>(electrocutionEntity);
 
+        // This shows up in the power monitor.
+        // Yes. Yes exactly.
+        MetaData(electrocutionEntity).EntityName = MetaData(uid).EntityName;
```

Comment:
Wow, this looks ugly but it turns out to just set `MetaDataComponent._entityName` so it's actually not that bad.